README.md

```diff
@@ -22,58 +22,57 @@ You first need to build the cli before building the docker images.
 
 ### Base image
 
-If you make changes to the [`src`](./src/) folder or the [`Dockerfile`](./Dockerfile), you need to rebuild the `containerbase/base` image.
+If you make changes to the [`src`](./src/) folder or the [`Dockerfile`](./Dockerfile), you must rebuild the `containerbase/base` image.
```

Comment:
pnpm build needs also be called

Reply:
```suggestion
If you make changes to the [`src`](./src/) folder or the [`Dockerfile`](./Dockerfile), you must:

- rebuild the `containerbase/base` image
- run `pnpm build`
```

Reply:
pnpm build first, then image build

Reply:
I made an ordered list and switched the order of the commands. 😉